lib/s3/s3-listener.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+const S3 = require('./s3-wrapper');
+
+class S3Listener {
+
+	constructor(event) {
+		this._event = event;
+	}
+
+	/**
+	 * Get the s3 bucket name
+	 *
+	 * @readonly
+	 * @memberof S3Listener
+	 */
+	get bucketName() {
+		return this._event.bucketName;
+	}
+
+	/**
+	 * Get the s3 object (file) key
+	 *
+	 * @readonly
+	 * @memberof S3Listener
+	 */
+	get fileKey() {
+		return this._event.fileKey;
+	}
+
+	/**
+	 * Get the s3 object (file) name
+	 *
+	 * @readonly
+	 * @memberof S3Listener
+	 */
+	get filename() {
```

Comment:
```suggestion
	get fileName() {
```
Me perturba que sean todos camelCase menos este y `fileSize`. Ajustate esos dos...